3.3/.sti/bin/assemble

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+#
+# Default STI assemble script for the python-3.3 image.
+#
+source "../lib/python_context"
```

Comment:
So that's another thing that we should agree on and do for other language images. 
@jhadvig @mnagy what do you guys say? I can do that for the other images as well :wink: